src/Tizen.Wearable.CircularUI.Forms.Renderer/Shell/NavigationDrawer.cs

```diff
@@ -0,0 +1,312 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Threading;
+using Xamarin.Forms;
+using ElmSharp;
+using ElmSharp.Wearable;
+using EWidget = ElmSharp.Widget;
+using ELayout = ElmSharp.Layout;
+using EColor = ElmSharp.Color;
+
+namespace Tizen.Wearable.CircularUI.Forms.Renderer
+{
+    public class NavigationDrawer : ELayout, IAnimatable
+    {
+        readonly int MinimumDrawerHeight = 50;
```

Comment:
It should be configurable by API

Reply:
`IconHeight` property has been added in `NavigationDrawer`.